appEnv.js

```diff
@@ -32,3 +32,7 @@ export const APP_BASE = IS_LOCAL
         ? process.env.NEXT_PUBLIC_VERCEL_PROJECT_PRODUCTION_URL
         : process.env.NEXT_PUBLIC_VERCEL_BRANCH_URL
     }`
+
+export const NEXT_PUBLIC_SEGMENT_WRITE_KEY = IS_PROD
```

Comment:
What the?

Can we _please_ put these in Vercel env variables before merging this? 🙏

Reply:
@oakinh https://vercel.com/good-party/gp-ui/settings/environment-variables